hieradata/common.yaml

```diff
@@ -22,7 +22,7 @@ deployable_applications: &deployable_applications
   - courts-api
   - courts-frontend
   - designprinciples
-  - EFG
+  - efg-training
```

Comment:
We should remove both apps from this list, right? Otherwise `efg-training` will show up as part of the "deploy app" job.